pipelines/pipelines/cli/build-branch.yaml

```diff
@@ -20,9 +20,15 @@ spec:
   - name: imageTag
     type: string
     default: main
-  - name: appname
+  - name: appnameCli
     type: string
     default: main-cli
+  - name: fromSimplatformBranch
+    type: string
+    default: prod
+  - name: appname
```

Comment:
I think just for clarity anywhere we have references to the CLI app and the maven repos app we should be really clear, so maybe call this 'appname-mavenrepos' or something?

Reply:
or appname-core, I don't mind